### ISSUE TITLE
TypeStatus

### ISSUE THREAD

**Issue by iphonecaptain** [w0lfschild/mySIMBL#28]
Keep in mind this is the first time I have ever installed a tweak on Mac.  I love it. Anyway, I HAVE installed mysimbl and easysimbl do i need both?  I am trying to get TypeStatus working on Mac.  If you can help I would appreciate it.  I can't understand how to disable System integrity protection.  Also it won't let me instal updates

**Comment by w0lfschild** [comment]
You need to disable System Integrity protection in order to install a critical component of mySIMBL.

http://www.macworld.com/article/2986118/security/how-to-modify-system-integrity-protection-in-el-capitan.html

**Comment by iphonecaptain** [comment]
My question is after I do this how do get the type status to work on Mac?  I have worked with this for hours

**Comment by iphonecaptain** [comment]
Do I leave this SIP off forever or just to install

**Comment by iphonecaptain** [comment]
Will you please help me. I've installed mysimbl and deleted easysimbl. Typestatus will not work still. I've installed it inside mysimbl but still nothing. Tell me what I'm doing wrong 

Mitch Stevens
http://www.iphonecaptain.com

**Comment by iphonecaptain** [comment]
My question is will typestaus work on os Sierra?  If so can you offer me any kind of help at all. I would really appreciate it

**Comment by w0lfschild** [comment]
It should work. I don't know what specific issue you're having is though.

**Comment by iphonecaptain** [comment]
I awn able to get doctor dark and others but type status does nothing

**Comment by iphonecaptain** [comment]
I tried contacting the dev but he basically said he didn’t know how to fix it either.  He acted like it wouldn’t working due to it needing an update.  Do you know anyone who has it working running Sierra

**Comment by w0lfschild** [comment]
Hmmm. It's working for me on 10.12.4 just fine.

Have you restarted messages since installing the plugin? Messages also has to be open for the indicator to appear.

**Comment by sammarcus** [comment]
Confirming it also works on my end, on 10.12.4 , (cc @hbang :) )

**Comment by sidpagariya** [comment]
Does it still work on macOS High Sierra? Also, if it doesn't work, where would you recommend looking for logs, etc.? I tried to inject the SIMBL script and this time around, it doesn't open all the apps that it used to inject into, so is that a fix or unexpected behavior?

Thanks,
Sid.

**Comment by sidpagariya** [comment]
Update: it still doesn't seem to work on High Sierra...at least for me.

Thanks!
Sid.

**Comment by Violet-01** [comment]
I have had it working on High Sierra previously. The issue I think is that due to whatever changes were made between 10.12 and 10.13 it has an issue interacting with menu bar. When it was working, it's because I installed on 10.12 and updated to 10.13. But after a restore it just doesn't seem to interact with the iMessage application and the menu bar the same way.

I messed around with it once a few months ago taking cues from message indicator that was/ is still working. Eventually got the welcome message to appear upon reopening the app but It would ultimately crash when someone did seem to start typing ( a message appearing on my phone right after ). I just have no idea how it was built and forgot about it.
